users/views.py

```diff
@@ -49,3 +51,26 @@ def post(self, request):
 
         except ValidationError as error:
             return JsonResponse({"message": error.message}, status=error.code)
+
+
+class SignInView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            email    = data['email']
+            password = data['password']
+
+            user     = User.objects.get(email = email)
+
+            access_token = jwt.encode({'id':user.id, 'exp': datetime.utcnow()+timedelta(days=1)},SECRET_KEY,algorithm=ALGORITHM)
```

Comment:
이렇게 수정하는 것이 조금 더 가독성이 더 좋은 것 같습니다.
```suggestion
            email    = data['email']
            password = data['password']
            user     = User.objects.get(email = email)
            
            payload = {
                'id'. :user.id, 
                'exp' : datetime.utcnow()+timedelta(days=1)
            }
            access_token = jwt.encode(payload,SECRET_KEY,algorithm=ALGORITHM)
```